lib/card_generator.rb

```diff
@@ -0,0 +1,21 @@
+require_relative 'helper'
+
+class CardGenerator
+  attr_reader :filename
```

Comment:
Since we aren't accessing `filename` outside of this class, we don't need an `attr_reader`